moabb/datasets/physionet_mi.py

```diff
@@ -120,17 +121,33 @@ def _get_single_subject_data(self, subject):
 
         # hand runs
         for run in self.hand_runs:
-            data['run_%d' % run] = self._load_one_run(subject, run)
+            raw = self._load_one_run(subject, run)
+            description_old = raw.annotations.description
+            description_new = []
+            for description in description_old:
+                if description == 'T0':
+                    description_new.append('rest')
+                elif description == 'T1':
+                    description_new.append('left_hand')
+                elif description == 'T2':
+                    description_new.append('right_hand')
+            raw.annotations.description = np.array(description_new)
+            data['run_%d' % run] = raw
+
```

Comment:
MOABB follows [PEP8](https://www.python.org/dev/peps/pep-0008/) code style convention, you could use flake8/pycodestyle/black to check/format your code.
You could remove this blank line.